wagn/lib/wagn/generators/wagn/templates/Gemfile

```diff
@@ -1,8 +1,8 @@
 source 'http://rubygems.org'
 
-gem 'wagn'<%=        ", :path=>'#{@wagn_path}/wagn'"  if @wagn_path.present? %>
-gem 'decko-rails'<%= ", :path=>'../wagn/decko-rails'" if @wagn_path.present? %>
-gem 'card'<%=        ", :path=>'#{@wagn_path}/card'"  if @wagn_path.present? %>, :require => false
+gem 'wagn'<%=        ", :path=>'#{@gem_path}/wagn'"  if @gem_path.present? %>
+gem 'decko-rails'<%= ", :path=>'#{@gem_path}/decko-rails'" if @gem_path.present? %>
```

Comment:
it's really just switching this relative path that is needed functionally, but it seems like "gem_path" is a more appropriate name here.